sdc/datatypes/common_functions.py

```diff
@@ -646,3 +649,60 @@ def _almost_equal_impl(x, y):
         return abs(x - y) <= numpy.finfo(common_dtype).eps
 
     return _almost_equal_impl
+
+
+def sdc_reindex_series(arr, index, name, by_index):
+    pass
+
+
+@sdc_overload(sdc_reindex_series, jit_options={'parallel': True})
```

Comment:
you don't need `jit_options={'parallel': True}`.
It overrides default behavior - based on environment variable

Reply:
Yes, that's valid point, I forgot to remove it. Let's fix that in following PR.